README.md

```diff
@@ -53,68 +39,37 @@ postgresql:
   service:
     nodePort: 30432
     type: NodePort
-ws:
-  image: loonride/funcx_ws
-  tag: latest
-  pullPolicy: Always
+websocketService:
+  image: funcx_websocket_service
+  tag: develop
+  pullPolicy: Never
 ```
 
-You can also clone this repository and build the Docker image if you want:
-
-```
-docker build -t funcx_ws:latest .
-```
-
-You should also use the latest dev versions of funcx-web-service and funcx-forwarder. You should deploy as usual like:
+You should deploy as usual like:
 
 ```
 helm install -f deployed_values/values.yaml funcx funcx
 ```
 
-Port 5000 will need to be exposed for the web service, and the WebSocket server uses port 6000. I expose these like this:
+Port 5000 will need to be exposed for the web service, and the WebSocket server uses port 6000:
 
 ```
 export POD_NAME=$(kubectl get pods --namespace default -l "app=funcx-funcx-web-service" -o jsonpath="{.items[0].metadata.name}") && kubectl port-forward $POD_NAME 5000:5000
-export POD_NAME=$(kubectl get pods --namespace default -l "app=funcx-funcx-ws" -o jsonpath="{.items[0].metadata.name}") && kubectl port-forward $POD_NAME 6000:6000
+export POD_NAME=$(kubectl get pods --namespace default -l "app=funcx-funcx-websocket-service" -o jsonpath="{.items[0].metadata.name}") && kubectl port-forward $POD_NAME 6000:6000
 ```
 
-Now, set up an endpoint, replace the endpoint UUID you received or chose with the one in the test script above, and run the test script.
-
-### How to set up and run the WebSocket server as a Python script without deploying it to a pod:
+## Run the WebSocket server as a Python script without deploying it to a pod
```

Comment:
I think this made sense when we started. Now, this is almost a crutch and indicated that the deployed side has shortcomings in terms of logging and debuggability.